metric_learn/_util.py

```diff
@@ -15,6 +15,14 @@ def vector_norm(X):
     return np.linalg.norm(X, axis=1)
 
 
+def has_installed_skggm():

```

Comment:
I don't think this needs to be a function, as the answer isn't going to change during execution.

Reply:
That's right thanks